### PR TITLE
feature: Starboard rewrite

### PR DESCRIPTION
**Pull request by DRSchlaubi**
- Rewrite Starboard.kt
- Add limit support

### CODE REVIEW

File: src/main/kotlin/com/github/devcordde/devcordbot/core/Starboard.kt [hunk 1]
```diff
@@ -110,94 +108,63 @@ class Starboard(private val starBoardChannelId: Long) {
...
-        val actualMessage = if (potentialEntryMessage == trackingMessage) {
-            event.guild.getTextChannelById(entry.channelId)?.retrieveMessageById(entry.messageId)?.await()
-        } else {
-            CompletableFuture.completedFuture(potentialEntryMessage).await()
-        }
-        if (actualMessage == null) {
-            deleteStarboardEntry(potentialEntryMessage.idLong, event.guild)
-            return
-        }
+        println("limmiting")
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/devcordde/devcordbot/core/Starboard.kt]: Print statement

File: src/main/kotlin/com/github/devcordde/devcordbot/core/Starboard.kt [hunk 2]
```diff
@@ -110,94 +108,63 @@ class Starboard(private val starBoardChannelId: Long) {
...
+        if (starrers.size >= limit) {
+            println("passed limit")
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/devcordde/devcordbot/core/Starboard.kt]: S.o.

**Review by jkoenig134 — CHANGES_REQUESTED** [review]
Look for print statements. Rest looks good.

**Comment by DRSchlaubi** [comment]
Looks like devcord killed my brain

**Review by jkoenig134 — APPROVED** [review]
LGTM

**Pull request merged**